docs/maintainer-guide.md

```diff
@@ -123,3 +124,22 @@ Ideally, only two branches should exist in the main repository:
 * `release` to contain the copy of the code running on the live server.
 
 The usage of any other branch should be accounted for, and the branches should be deleted as soon as they are no longer needed.
+
+### Community membership
+
+To elect a new committer:
+
+* Add the GitHub user to the `Committers` team.
+* Add the committer's name and photo to `about.jsp`.
```

Comment:
Write access is part of "Committers" team. Owner access is an organization-wide permission; "Team-leads" team grants only admin access.